### ISSUE TITLE
Creating constructors for static attributes and no attributes

### ISSUE THREAD

**Issue by PedroLuisDionisioFraga** [sohibegit/vscode-java-code-generator#59]
Good morning, I noticed a problem when I create a constructor for my class when it has no attributes or only static attributes, the constructor misses the last letter of the class name and the first parenthesis, example.

```
public class Converter {
  public static int id = 0;

  public static int toConvert(int points) {
    return points * 100;
  }

  // Building a empty constructor
  public Converter() {
  }
  // Building a constructor with all fields
  public Converte) {
  }

}
```

**Comment by sohibegit** [comment]
fixed with v 8.0.0

**Comment by PedroLuisDionisioFraga** [comment]
The version already ready? Or is a future correction?
